scripts/pico_project.py

```diff
@@ -870,19 +858,19 @@ def generateProjectFiles(projectPath, projectName, sdkPath, projects, debugger,
     "cmake.cmakePath": "{cmakePath.replace(user_home, "${userHome}") if use_home_var else cmakePath}",
     "C_Cpp.debugShortcut": false,
     "terminal.integrated.env.windows": {{
-        "PICO_SDK_PATH": "{propertiesSdkPath(sdkVersion, force_windows=True)}",
-        "PICO_TOOLCHAIN_PATH": "{propertiesToolchainPath(sdkVersion, force_windows=True)}",
-        "Path": "{propertiesToolchainPath(toolchainVersion, force_windows=True)}/bin;{os.path.dirname(cmakePath.replace(user_home, "${env:USERPROFILE}") if use_home_var else cmakePath)};{os.path.dirname(ninjaPath.replace(user_home, "${env:USERPROFILE}") if use_home_var else ninjaPath)};${{env:PATH}}"
+        "PICO_SDK_PATH": "{codeSdkPath(sdkVersion)}",
+        "PICO_TOOLCHAIN_PATH": "{codeToolchainPath(sdkVersion)}",
+        "Path": "{codeToolchainPath(toolchainVersion)}/bin;{os.path.dirname(cmakePath.replace(user_home, "${userHome}") if use_home_var else cmakePath)};{os.path.dirname(ninjaPath.replace(user_home, "${userHome}") if use_home_var else ninjaPath)};${{env:PATH}}"
     }},
     "terminal.integrated.env.osx": {{
-        "PICO_SDK_PATH": "{propertiesSdkPath(sdkVersion, force_non_windows=True)}",
-        "PICO_TOOLCHAIN_PATH": "{propertiesToolchainPath(toolchainVersion, force_non_windows=True)}",
-        "PATH": "{propertiesToolchainPath(toolchainVersion, force_non_windows=True)}/bin:{os.path.dirname(cmakePath.replace(user_home, "${env:HOME}") if use_home_var else cmakePath)}:{os.path.dirname(ninjaPath.replace(user_home, "${env:HOME}") if use_home_var else ninjaPath)}:${{env:PATH}}"
+        "PICO_SDK_PATH": "{codeSdkPath(sdkVersion)}",
+        "PICO_TOOLCHAIN_PATH": "{codeToolchainPath(toolchainVersion)}",
+        "PATH": "{codeToolchainPath(toolchainVersion)}/bin:{os.path.dirname(cmakePath.replace(user_home, "${userHome}") if use_home_var else cmakePath)}:{os.path.dirname(ninjaPath.replace(user_home, "${userHome}") if use_home_var else ninjaPath)}:${{env:PATH}}"
     }},
     "terminal.integrated.env.linux": {{
-        "PICO_SDK_PATH": "{propertiesSdkPath(sdkVersion, force_non_windows=True)}",
-        "PICO_TOOLCHAIN_PATH": "{propertiesToolchainPath(toolchainVersion, force_non_windows=True)}",
-        "PATH": "{propertiesToolchainPath(toolchainVersion, force_non_windows=True)}/bin:{os.path.dirname(cmakePath.replace(user_home, "${env:HOME}") if use_home_var else cmakePath)}:{os.path.dirname(ninjaPath.replace(user_home, "${env:HOME}") if use_home_var else ninjaPath)}:${{env:PATH}}"
+        "PICO_SDK_PATH": "{codeSdkPath(sdkVersion)}",
+        "PICO_TOOLCHAIN_PATH": "{codeToolchainPath(toolchainVersion)}",
+        "PATH": "{codeToolchainPath(toolchainVersion)}/bin:{os.path.dirname(cmakePath.replace(user_home, "${userHome}") if use_home_var else cmakePath)}:{os.path.dirname(ninjaPath.replace(user_home, "${userHome}") if use_home_var else ninjaPath)}:${{env:PATH}}"
```

Comment:
Actually I've had a test and the integrated terminal paths still need to have platform specific ${env:HOME}, as the ${userHome} doesn't seem to work for those paths. Could we therefore keep the propertiesXXXPath (maybe rename them to terminalXXXPath) functions, so they can still be used in the integrated terminal environments?

Reply:
Right, i'll fix that.